README.md

```diff
@@ -1,63 +1,98 @@
+4. Copy the `REVERSED_CLIENT_ID` from the same plist file into line 39 in `./mobile/app.json`.
+5. Download the `google-services.json` file from the android app on firebase. Place this file in `./mobile/configs/app` and in `./mobile/android/app`.
+6. Copy the `current_key` and `mobilesdk_app_id` from the json file you just downloaded into the android object in the `FirebasePlatformOverrides` variable in `./config/app.js`. Then copy the `client_id` on line 18 to line 47 of the same `app.js` file.
+7. Again from the downloaded json file, copy the `current_key` and the `certificate_hash` to lines 54 and 55 of `./mobile/app.json`.
+8. Create `.runtimeconfig.json` in `./server/functions` with the following:
+
+	```
+	{
+	  "envs": {
+	    "project_name": "[appName]",
+	    "sendgrid_api_key": "...",
+	    "sendgrid_emails_validation_api_key": "...",
+	    "is_prod": "false", // set false for staging,
+	    "devlogin": true,
+	    "twilio_phone_number_from": "...",
+	    "twilio_auth_token": "...",
+	    "twilio_account_sid": "..."
+	  }
+	}
+	```
+9. Add the Sendgrid API key to the `.runtimeconfig.json` file you just created (get this key from a dev on the team).
+10. Open the `./mobile/ios` folder in Xcode. Go to the "Info" tab and scroll down to "URL Types". Add a new URL type with a "Role" of "Editor" and in "URL Schemes" paste in the `REVERSED_CLIENT_ID` from the plist file you downloaded earlier.
+
+
+## Run Setup Script
 
 Navigate back to the root directory and run:
+
 ```
-yarn all
+./bin/setup.bash
 ```
 
-### Configuration files references
+## Running the App
 
-Environment config file `./server/functions/.env` (should be created if not exists).
-    * .env file content `GOOGLE_APPLICATION_CREDENTIALS=/path/to/json/credentials`
-   
-### Validate fields in app.json
+### Run & build app
 
-1. ios.bundleIdentifier should be ```"com.maslo.evolution"```
-2. ios.config.googleSignIn.reservedClientId should be ```com.googleusercontent.apps.858064288439-02l37uab6frvq67228g5taua7c112pii```
-3. android.package should be ```"com.maslo.evolution"```
-4. android.config.googleSignIn.apiKey should be ```AIzaSyBbEpbXQY-xKsrMzrsdXNBuBdHzsTpPMBw```
-5. android.config.googleSignIn.certificateHash should be ```858064288439-oq1lit38a9cc39sb6dhcrrjmmk8k8i6t.apps.googleusercontent.com```
+* **IOS**: To run project locally run `yarn ios` from the `./mobile` directory
+* **Android**: To run project locally the first time, open the android directory in android studio, sync with gradle and run the app. In subsequent runs, run `yarn android` from the `./mobile` directory
 
+## Maslo Dashboard
+
+The staging dashboard is currently hosted [here](https://bipolarbridges.web.app/)
+
+## Redeploy Cloud Functionality and Dashboard
```

Comment:
@cara-wong I think we need a step to setup the `.env` file containing the firebase credentials path

Reply:
My new project somehow deployed without it, but I think that's because my firebase tools are already configured.

Reply:
From my understanding the .env file with the GOOLE_APPLICATION_CREDIENTIALS is for providing authentication credentials to the app with a service account (for testing). It's not necessary to the functionality of the app. I don't believe it's something that's configured in firebase, rather a service account is created in the Google Cloud Platform and we can download the json file from that service account and point to it in the .env file if we want to authenticate using it. That's why everything works fine without the .env file. This step doesn't hurt to do, so I can definitely add it if you think it's valuable. @crugwiro What is your understanding of the .env file and the google credentials variable?

Reference: https://cloud.google.com/docs/authentication/getting-started

Reply:
Added .env file instructions